src/cli/args.js

```diff
@@ -1,5 +1,14 @@
 const parseArgs = () => {
-    // Write your code here 
+
+  // getting args only (omit "node src/cli/args.js")
+  const args = process.argv.slice(2);
+
+  for (let i = 0; i < args.length; i += 2) {
+    // omit the "--" from arg
+    const propName = args[i].slice(2);
+    const value = args[i + 1];
+    console.log(`${propName} is ${value}`);
```

Comment:
Regarding requirements - the output should be comma separated, not a new line
Anyway, I didn't reduce the points amount for this task.